src/collider/src/collider.cpp

```diff
@@ -154,15 +154,13 @@ struct CreateRadixTree {
 
 template <typename T, const bool selfCollision, typename Recorder>
 struct FindCollisions {
+  VecView<const T> queries;
   VecView<const Box> nodeBBox_;
   VecView<const thrust::pair<int, int>> internalChildren_;
```

Comment:
Thinking if we can remove this `thrust::pair` as well, maybe with `std::pair`?
Not really related to fancy iterator, but I feel that the reason for using `thrust::pair` is mostly because of `zip`.